### DISCUSSION TITLE
Can a tenant have multiple sub-tenants? (Parent-Child hierarchy)

### DISCUSSION THREAD

**Discussion by nurkmez2** [abpframework/abp#17273] [Ideas]
Hi @hikalkan 
It would be great if the following Parent-Child hierarchy is supported by default instead of a one-level structure

**-Host-1** 
 **--Parent-Tenant-01**
 ------Tenant-01-Sub
 ------Tenant-02-Sub
 **--Parent-Tenant-02**
 ------Tenant-02-Sub
 ------Tenant-02-Sub
  Thanks 
  Best Regards

**Comment by ahmadalmunajjed** [comment]
Hi @nurkmez2, I am curious to know 
- what the business requirements are? 
- and what is your approach of implementing it 
- and how this Parent-Child hierarchy would be useful
- would having a type to tenant helps?

**Comment by Onoscity** [comment]
Yes
